tests/sqlparser_clickhouse.rs

```diff
@@ -220,6 +220,88 @@ fn parse_create_table() {
     );
 }
 
+#[test]
+fn parse_create_table_with_primary_key() {
+    match clickhouse().one_statement_parses_to(
```

Comment:
per the generic dialect comment earlier, if the dialect allows, then we can change this to `clickhouse_and_generic()`

Reply:
Updated.